service/src/main/resources/logback.xml

```diff
@@ -1,14 +1,30 @@
 <configuration>
-    <conversionRule conversionWord="coloredLevel" converterClass="play.api.libs.logback.ColoredLevel" />
-
-    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder>
-            <pattern>%date{HH:mm:ss.SSS} %highlight([%-5level]) %cyan(%logger{16}) [%thread] - %message%n%xException{10}</pattern>
+    <appender name="JSON-STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder class="net.logstash.logback.encoder.LoggingEventCompositeJsonEncoder">
```

Comment:
Maybe `LogstashEncoder` is enough for our needs now.

Reply:
Donesn't hurt now + more flexibility later on when we use MDC for correlation id in the future.